src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx

```diff
@@ -0,0 +1,81 @@
+  src={slack_auth}
+/>
+
+Once a workspace is configured, any member of your New Relic organization will be able to authenticate to that workspace and share new discussion threads to Slack. It's important to note that before new users can sync with Slack, a New Relic account admin needs to complete the sync first.
+
+If your organization has more than one account, then Slack credentials are tied to each specific account. If you are trying to sync Slack with a different account, then Slack will require you to authenticate again. This feature was created for security purposes as some multi-org accounts require this extra layer of protection. We are working on solutions and options to make it more convenient.
+
+## Syncing to Slack [#sync]
+
+After authenticating with a Slack workspace, when starting a new discussion thread (or adding a comment to a thread that hasn't previously been sync'd to Slack) simply check the “Sync to” checkbox and select the Slack channel you like to post the comment to.  
+
+<img
+  title="Slack channel"
+  alt="A screenshot of a communication thread that will be synced with Slack."
+  src={slack_channel}
+/>
+
+After submitting a comment, the thread will have a “Synced to Slack” status indicator at the top letting all teammates know that any comments added to the thread on New Relic will also be added to the thread on Slack.
+
+Each new thread or discussion can be synced to a different Slack channel. This includes starting new discussions on different pages or starting additional discussions on pages that have previously synced discussions.
+
+After a discussion is synced to Slack, you won't be able to turn off the sync. This is to ensure that New Relic and Slack are continuously aligned. This eliminates confusion, fragmented conversations, and lost information. If the sync from New Relic to Slack is turned off, it would allow some teammates to continue the discussion on New Relic and others to continue on Slack. You can, however, start a new discussion on the same page and elect not to sync that new discussion if you wanted to discuss something just on New Relic but not Slack.
+
+<img
+  title="Synced Slack"
+  alt="A screenshot of a successful Slack integration."
+  src={slack_synced}
+/>
+
+## Replying from Slack [#reply]
+
+The first comment in a thread synced to Slack will be added to the channel selected on New Relic. The Slack post will include an “Open in New Relic” button providing quick navigation to the page and discussion on [one.newrelic.com](one.newrelic.com).
+
+<img
+  title="Slack reply thread"
+  alt="A screenshot of a Slack reply thread."
+  src={post_on_slack}
+/>
+
+The post will also include a message indicating the thread is synced to New Relic and that replies added to the thread on Slack will appear on New Relic as well.  Please note this means that regardless of the Slack channel's member list **replies will be visible by everyone in your New Relic organization on newrelic.com**.
```

Comment:
For the last sentence, what do you think about switching it around again to "Please note that this means that replies will be visible by everyone in your New Relic organization on newrelic.com regardless of the Slack channel's member list." ?